### ISSUE TITLE
Derived datasets: Remove dummy scaling attributes from DerivedDataset class

### ISSUE THREAD

**Issue by ploh** [quintel/atlas#82]
Currently, the scaling attribute is defined with
```
attribute :scaling,      Preset::Scaling
```
in `DerivedDataset`. But since the Preset::Scaling has attributes `has_energy`, `has_agriculture` and `has_industry` which are not respected when set for a DerivedDataset#scaling, this is not ideal - also, it is unnecessary, since the DerivedDataset has all the attributes of a Dataset directly, among them a lot of `has_...` attributes (`has_energy` is missing in the attributes of `Dataset`. Maybe we should add that? @antw @ChaelKruip).

Solution:
The `Preset::Scaling` class should get a newly invented superclass `Atlas::Scaling` without the `has_...`. This superclass is then used for the DerivedDataset#scaling.

**Comment by ChaelKruip** [comment]
@ploh why am I assigned for this ticket? I have no idea what to do with this...

Re-assigning @ploh

**Comment by ploh** [comment]
@ChaelKruip Yes, sorry, I should have commented why I did that. This is, of course, not for you to implement, but for @grdw - who is also assigned. I assigned it to you as well because I thought you are probably the one to decide / prioritize when/if Gerard should do this.
